google/cloud/firestore_v1/document.py

```diff
@@ -513,8 +514,8 @@ def on_snapshot(self, callback):
             collection_ref = db.collection(u'users')
 
             def on_snapshot(document_snapshot, changes, read_time):
-                doc = document_snapshot
-                print(u'{} => {}'.format(doc.id, doc.to_dict()))
+                for doc in collection_snapshot.documents:
```

Comment:
Arg name should be changed as well
![image](https://user-images.githubusercontent.com/26030071/76759197-45184900-679c-11ea-9d6b-800f22774b43.png)